### PR TITLE
fix fork-rename.sh OSX

### PR DESCRIPTION
**Pull request by maleck13**
The script did not work as expected on OSX Due to differences in the sed command.

### CODE REVIEW

**Comment by maleck13** [comment]
ping @bparees quick fix for the rename script

File: hack/fork-rename.sh [hunk 1]
```diff
@@ -5,7 +5,12 @@ if [[ -z $1 || -z $2 ]]; then
...
-
-sed -i "s#github.com/openshift/open-service-broker-sdk#github.com/$1/$2#g" Makefile
+OS_TARGET=`uname -s`  
+BACKUP=""
+if  [ "$OS_TARGET" == "Darwin" ]; then
+  BACKUP="''"
+fi
+  
+find . -type f -name *.go -exec sed -i $BACKUP  "s#github.com/openshift/open-service-broker-sdk#github.com/$1/$2#g" {} +
+  sed -i $BACKUP  "s#github.com/openshift/open-service-broker-sdk#github.com/$1/$2#g" Makefile
```
Comment (bparees) [inline hack/fork-rename.sh]: drop this extra space?

**Review by bparees — CHANGES_REQUESTED** [review]

**Comment by maleck13** [comment]
hmm just noticed a bug here. It is creating a backup with a name such as ```Makefile''``` for OSX so have switched it be a full os specific command . Unfortunately not a bash wizard. Maybe there is a better way?

**Comment by bparees** [comment]
good enough for now.

**Pull request merged**